pkg/v1/tkg/client/utils.go

```diff
@@ -425,3 +427,70 @@ func getCCPlanFromLegacyPlan(plan string) (string, error) {
 	}
 	return "", errors.Errorf("unknown plan '%v'", plan)
 }
+
+func (c *TkgClient) getAutoScalerValuesFileFromConfigs(options *CreateClusterOptions) (string, error) {
+	return c.GetAutoScalerValuesFileFromConfigs(options.ClusterName, options.TargetNamespace, "workload", options.ProviderRepositorySource.InfrastructureProvider)
+}
+
+func (c *TkgClient) GetAutoScalerValuesFileFromConfigs(clusterName, namespace, clusterRole, infrastructureProvider string) (string, error) {
+	autoScalerMap := map[string]string{
+		constants.ConfigVariableClusterName:  clusterName,
+		constants.ConfigVariableNamespace:    namespace,
+		constants.ConfigVariableClusterRole:  clusterRole,
+		constants.ConfigVariableProviderType: infrastructureProvider,
+	}
+
+	autoscalerMaxNodesTotal, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableAutoscalerMaxNodesTotal)
```

Comment:
we don't do any sanity checks on these values do we? someone could do `AUTOSCALER_MAX_NODES_TOTAL=banana` right? It might save us headaches later on to add some sanity checks.